README.md

```diff
@@ -56,7 +56,7 @@ included will not be changed from the defaults.  `enabled` is set to false by de
 
 ### States
 
-#### `enable()`
+#### `enable`
```

Comment:
I think we accidentally deleted the parentheses here. We should prob keep them to be consistent with the rest of the methods